src/main/scala/diplomaticobjectmodel/model/OMRocketCore.scala

```diff
@@ -58,7 +58,7 @@ object OMCaches {
 
   def icache(p: ICacheParams, resourceBindings: Option[ResourceBindings]): OMICache = {
     OMICache(
-      memoryRegions = resourceBindings.map(DiplomaticObjectModelAddressing.getOMMemoryRegions("ICache", _)).getOrElse(Nil),
+      memoryRegions = resourceBindings.map(DiplomaticObjectModelAddressing.getOMMemoryRegions("ITIM", _)).getOrElse(Nil),
```

Comment:
Is the implication that the ICache only has a memory region if it's configured as an ITIM? Is there a similar issue above with the DCache/DTIM?

Reply:
Yes to the first question and no to the second. When an I-Cache is (Chisel-time) configured as an ITIM, it allows software to lock some of the cache ways at runtime. Note that not all of the ways can be locked, so part of the I-Cache must always function like a cache rather than a TIM.

The D-Cache/DTIM does not function this way, since it is only a Chisel-time configuration and not a software runtime configuration. The DTIM is also configured fully as a scratchpad and does not have the same way-locking functionality as the ITIM.

Reply:
Thanks for the clarification!